src/components/Dropdown/__stories__/dropdown.stories.mdx

```diff
@@ -18,14 +14,36 @@ import "./dropdown.stories.scss"; import {
   StoryDescription
 } from "../../../storybook/components/story-description/story-description";
 
+export const metaSettings = createStoryMetaSettings({
+    component: Dropdown,
+    enumPropNamesArray: ["size"],
```

Comment:
ok, it sounds like that if `enumPropNamesArray` could receive a string OR an object (in the form of `{propName: "size", options: DropDown.size}`, would be helpful. What do you think? (in any case, it can be done outside of this PR 😄  )

Reply:
great idea, will be done in this iteration :)